tests/test_transforms/test_math/test_statistics_transform.py

```diff
@@ -177,6 +210,7 @@ def test_std_feature(simple_df_for_agg: pd.DataFrame, window: int, periods: int,
         MeanTransform(in_column="target", window=5),
         StdTransform(in_column="target", window=5),
         StdTransform(in_column="target", window=5),
+        MADTransform(in_column="target", window=5),
     ),
 )
 def test_fit_transform_with_nans(transform, ts_diff_endings):
```

Comment:
This is a method was created to check that transforms simply don't fail working with NaNs. May you also create test that checks that this transform handles NaNs correctly.

Reply:
done